src/ucm/event/event.c

```diff
@@ -621,6 +621,11 @@ ucs_status_t ucm_test_events(int events)
     return ucm_mmap_test_installed_events(events);
 }
 
+ucs_status_t ucm_test_external_events(int events)
+{
+    return ucm_mmap_test_events(events & ucm_external_events, 1);
```

Comment:
can pass the string "external" to ucm_mmap_test_events() instead of 1

Reply:
done